docs/permissions/plugin-authors/02-adding-a-basic-permission-check.md

```diff
@@ -285,4 +285,44 @@ Then we want to update the `plugins/todo-list-backend/src/service/standaloneServ
   module.hot?.accept();
 ```
 
+Finally, we need to update `plugins/todo-list-backend/src/plugin.ts`:
```

Comment:
I guess this backend system file has been added to the example plugin since the last time we ran through the tutorial, and it needs some changes once permissions support is added.